src/PowerAuth.ts

```diff
@@ -528,6 +529,26 @@ export class PowerAuth {
         return new PowerAuthPassword(destroyOnUse, onAutomaticCleanup, this.instanceId)
     }
 
+    // End-To-End Encryption
+
+    /**
+     * Creates a new instance of encryptor suited for application's general end-to-end encryption purposes.
+     * The returned encryptor is cryptographically bounded to the PowerAuth configuration, so it can be used
+     * with or without a valid activation.
+     */
+    getEncryptorForApplicationScope(): PowerAuthEncryptor {
+        return new PowerAuthEncryptorImpl('APPLICATION', this.instanceId)
+    }
+
+    /**
+     * Creates a new instance of encryptor suited for application's general end-to-end encryption purposes.
```

Comment:
This line is probably copy-pasted from the application scope and should have different wording.

Reply:
The wording is OK, because the scope is explained in the next sentence. I just removed `application's` from the description of both functions, to make it more clear.